how-to/networking/chrony-client.md

```diff
@@ -3,34 +3,82 @@
 
 Ubuntu uses `chrony` for synchronizing time, which is installed by default as of Ubuntu 25.10. You can optionally use `timedatectl`/`timesyncd` to {ref}`synchronize time using systemd <timedatectl-and-timesyncd>`.
 
-In this guide, we will show you how to configure these services.
+In this guide, we will show you how to configure this service.
```

Comment:
```suggestion.
```

Reply:
I'd remove this line, it's a bit ambiguously worded and if we clarify it, it's basically a repetition of the first line. Better to just get rid of it :)